starter/js/atm.js

```diff
@@ -1,38 +1,99 @@
 //Begin with the document ready function
+      $(document).ready(function() {
+        document.body.style.backgroundColor = "red";
+  //On click of the depositChecking button
+      $("#depositChecking").click(function() {
+        depositChecking();
+      });
+  //On click of the withdrawChecking button
+      $("#withdrawChecking").click(function() {
+        withdrawChecking();
+      });
+  //On click of the depositSavings button
+      $("#depositSavings").click(function() {
+        depositSavings();
+      });
+  //On click of the withdrawlSavings button
+      $("#withdrawSavings").click(function() {
+        withdrawSavings();
+      });
+});
+
+  // var checkingBalance = 0;
+
+//Checking account deposit function
+
+  function depositChecking() {
+    //Get value from the amountChecking input field
+    var amount = $("#amountChecking").val();
+
+    checkingBalance += amount;
+
+    $("#checkingBalance").html("$" + amount);
+
+};
+
+
+
+
 
-    //Checking account deposit function
 
-      //On click of the depositChecking button
 
-        //Get value from the amountChecking input field
 
         //Take that value and add it to the existing value in the checkingBalance div
 
-    //Checking account withdrawl funtion
+//Checking account withdrawl funtion
+  function withdrawChecking() {
+    //Get value from the amountChecking input field
+    var amount = $("#amountChecking").val();
+    //Take that value and add it to the existing value in the checkingBalance div
+    checkingBalance -= amount;
 
-      //On click of the withdrawChecking button
+    if (amountChecking > checkingBalance) {
```

Comment:
Make sure your variables are consistent. I think you meant amount here and not amountChecking.